src/mlpack/bindings/python/ConfigureSetup.cmake

```diff
@@ -17,30 +17,4 @@
 # - MLPACK_LIBDIR: path to mlpack libraries
 # - OUTPUT_DIR: binary output directory for CMake
 
-# It's possible that the FindBoost CMake script may have returned a Boost
```

Comment:
I guess we can remove this file too, and alter the changes accordingly for setup.py.in, just an idea may be I am missing something here.

Reply:
May be we can't remove this file directly, but we can use other file here.

https://github.com/mlpack/mlpack/blob/11f60ee7be17225d350622bb3c0dd65ac8558882/src/mlpack/bindings/python/CMakeLists.txt#L31

We can use:
```cmake
-P "${CMAKE_SOURCE_DIR}/CMake/ConfigureGenerate.cmake"
```
and here.
https://github.com/mlpack/mlpack/blob/11f60ee7be17225d350622bb3c0dd65ac8558882/src/mlpack/bindings/python/CMakeLists.txt#L18-L19

```CMake
-D GENERATE_CPP_IN=${CMAKE_SOURCE_DIR}/src/mlpack/bindings/python/setup.py.in
-D GENERATE_CPP_OUT=${CMAKE_BINARY_DIR}/src/mlpack/bindings/python/setup.py
```

Reply:
@Yashwants19 Perfect I understand. I have already removed the file, so do I need to bring this file back or not?

Reply:
I agree with @Yashwants19 here; now there's no need for special handling.  I think we should remove `ConfigureSetup.cmake`, and then *also* rename `ConfigureGenerate.cmake` to `ConfigureFile.cmake` and rename the arguments accordingly (since it is no longer specific to configuring C++ files), and then change all uses of `ConfigureGenerate.cmake` to `ConfigureFile.cmake`. :+1:

Reply:
Sorry, I was looking at the wrong diff again, so it looks like `ConfigureSetup.cmake` is already gone, which is great, so my only additional suggestion would be to fix the name and comments inside of `ConfigureGenerate.cmake` so that it is `ConfigureFile.cmake`. :+1: